pkg/pxc/app/statefulset/proxysql.go

```diff
@@ -462,6 +482,28 @@ func (c *Proxy) PMMContainer(ctx context.Context, cl client.Client, spec *api.PM
 	return &ct, nil
 }
 
+// pmm3ProxySQLEnvVars returns a list of environment variables to configure the PMM3 container for monitoring proxysql.
+func pmm3ProxySQLEnvVars(pmmProxysqlParams string) []corev1.EnvVar {
+	return []corev1.EnvVar{
+		{
+			Name:  "DB_TYPE",
+			Value: "proxysql",
+		},
+		{
+			Name:  "PMM_ADMIN_CUSTOM_PARAMS",
+			Value: pmmProxysqlParams,
+		},
+		{
+			Name:  "DB_ARGS",
+			Value: "--dsn $(MONITOR_USER):$(MONITOR_PASSWORD)@tcp(localhost:6032)/",
```

Comment:
Does it work if `MONITOR_USER` or `MONITOR_PASSWORD` uses special characters like `@` or `:`?